specifications/api/swagger.yaml

```diff
@@ -2790,6 +2930,16 @@ x-responsesPostExternalTransaction: &responsesPostExternalTransaction
       application/json:
         schema: *ApiTxId
 
+x-responsesPostAnyAddress: &responsesPostAnyAddress
+  <<: *responsesErr400
+  <<: *responsesErr406
+  <<: *responsesErr415
+  202:
```

Comment:
As far as I see it is actually `201` on success.

Reply:
nice catch, will make it PostAccepted rather than PostCreated and it will return not 201 but 202as advertised in swagger.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/3d003584ca7245da6c5f7012b712a4880d7955be